CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 11.3.2
+ - Refactor: review manticore error handling/logging [#1029](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1029)
```

Comment:
I would restate as
```suggestion
 - Refactor: review manticore error handling/logging, logging originating cause in case of connection related error when debug level is enabled [#1029](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1029)
```